ajax/libs/country-region-dropdown-menu/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "country-region-dropdown-menu",
   "filename": "geodatasource-cr.min.js",
-  "version": "1.2.2",
+  "version": "2.1.1",
```

Comment:
This change needs rollback, thanks.

Reply:
Hi @PeterDaveHello 
You mean require us to rollback?

Reply:
Hi @PeterDaveHello,

Any update on this?